source/agora/test/Quorum.d

```diff
@@ -38,84 +38,57 @@ import core.time;
 ///
 unittest
 {
-    TestConf conf = {
-        outsider_validators : 2,
-        extra_blocks: GenesisValidatorCycle - 2 };
+    TestConf conf = { outsider_validators : 2,
+        txs_to_nominate : 0 };
```

Comment:
Hmm. Can you remind me what the effect of setting it to zero is?

Reply:
It is unlimited. In a PR coming soon this will be the default in the Base class.

Reply:
Added comment